<?xml version="1.0"?>

<launch>
  <param name="id_joint_table" value="$(find thormang3_demo)/data/id_joint_table.yaml"/>
  
  <!-- thormang3 GUI demo kit -->
  <node name="thormang3_demo_opc" pkg="thormang3_demo" type="thormang3_demo" output="screen">
    <remap from="/thormang3_demo/ik_target_pose" to="/pose_panel/pose" />
  </node>
  
  <!-- step generator for walking demo -->
  <node name="thormang3_foot_step_generator" pkg="thormang3_foot_step_generator" type="thormang3_foot_step_generator_node" output="screen"/>
</launch>
